### ISSUE TITLE
Validation Tool - Notify User when Comparing Model vs Predictions

### ISSUE THREAD

**Issue by jamesdoyle21** [asascience/ott-coops#435]
OceansMap Bug Tracker line 63

This is obviously bad because we are validating a prediction with a prediction. The solution is to not allow the user to select these stations in the validation tool. If the user really wants to see both plotted on the same graph, they can do so in the obs drawer.

This relates to #434 because if we can differentiate these stations on the map, it should be easier to remove them from the validation tool.

![image](https://user-images.githubusercontent.com/82955395/163228422-b09d1f35-1395-43a2-8cfc-c313fdf69d66.png)

**Comment by mpiannucci** [comment]
I disagree with this one... did chris specify this?

**Comment by jamesdoyle21** [comment]
@mpiannucci this came from Jeff. What are your thoughts here? Perhaps a notification to let the user know they are comparing a model to a station prediction?

**Comment by mpiannucci** [comment]
Yeha in theory kelly had mentioned being able to validate models against other models, so i see nothing wrong with validating against predictions IF and ONLY IF there are no actual measurements available for that given station

**Comment by jamesdoyle21** [comment]
I agree with that. As long as we make note of it so the user isn't unknowingly assuming it's measured data. An orange notification in the same place we put the "error no data" should do it

**Comment by cpaternostro** [comment]
There are often times when I want to compare model v pred.  Likely because
no other data exists.
But in general, this tool should allow anything to be able to be compared
to anything.  The user needs to be aware of GIGO.


--

Chris Paternostro

https://tidesandcurrents.noaa.gov/

**Comment by jamesdoyle21** [comment]
This issue came up when looking at Tide Predictions in the validation tool. I am getting no comparison options when I select tide predictions as the parameter. Is this a separate issue?

I also cannot use a station with only tide predictions as a comparison for water level - it spits an error.

![image](https://user-images.githubusercontent.com/82955395/163843699-790be28b-272b-4855-92b9-40e07afd6494.png)

**Comment by jamesdoyle21** [comment]
@kvmartin001 do you have a test case for this issue?

**Comment by kvmartin001** [comment]
@jamesdoyle21 No, in my PR, Tide Predictions was supposed to be stripped as an option in that field completely, so the case above is a bug.
I could fix the bug with a one-line change, but I think we want to just allow Tide Predictions again, which would make the bug a moot point.

**Comment by kvmartin001** [comment]
Tide predictions are enabled again (https://github.com/asascience/oceansmap-3/pull/356), so I moved this to Merged. Do we still want a notification?

Perhaps inline like this?
![image](https://user-images.githubusercontent.com/56608780/163863659-70bc3194-6085-4810-8caf-0bde074093bd.png)

**Comment by jamesdoyle21** [comment]
Yeah, I like that

**Comment by jamesdoyle21** [comment]
If the user selects "Tide Predictions" as the parameter for the validation tool, is there a comparison model that is supposed to populate? Lets plan to discuss this in the meeting on Tuesday as well

**Comment by mpiannucci** [comment]
Any water level

**Comment by jamesdoyle21** [comment]
I am still getting no comparison models when I select Tide Predictions

![image](https://user-images.githubusercontent.com/82955395/168116782-6ce7807b-e574-4bc7-9b65-2ce2c1cbf26b.png)

**Comment by jamesdoyle21** [comment]
Also, this ties into #434 but when the user uses the "Show Available Stations" button, the map is populated with all the stations that have water level listed, but will never have water level data available. And is the vast majority of these stations showing

![image](https://user-images.githubusercontent.com/82955395/168117378-e1a2ffdc-bcc0-4247-a27f-f42582541e89.png)

**Comment by mpiannucci** [comment]
I mean there's not really a way to filter based on that unless we get more information from the catalog. We only know what timesteps are available for the station, not which timesteps have what kind of data (right now)
